__mocks__/Accounts.ts

```diff
@@ -51,9 +51,9 @@ export const simpleWallet2 = SimpleWallet.createFromPrivateKey(
 );
 
 export const WalletsModel1: AccountModel = {
-    id: 'someId',
+    id: 'WalletsModel1',
     node: '',
-    profileName: 'profile_name',
+    profileName: 'profile1',
```

Comment:
I'd prefer that we have clearer naming throughout. 'WalletsModel2' and 'Profile1' is the naming equivalent of 'Final Edit_final_final_v2.docx'.

Reply:
these names were inherited so didn't want to change, you're right, noted 👍